gradle.properties

```diff
@@ -2,6 +2,9 @@ VERSION_NAME=1.7.0
 VERSION_CODE=0
 GROUP=com.soundcloud.lightcycle
 
+android.useAndroidX=true
+android.enableJetifier=true
```

Comment:
will this still work if we disable the jetifier? i think that would be preferred.

Reply:
not sure why we want to disabling it? is there a particular reason?

I just followed the migration guide 
https://developer.android.com/jetpack/androidx/migrate

Reply:
can we disable it? if disabled, will it compile?

Reply:
works fine if set this flag to `false`. do you want to have it disabled?

Reply:
false would be preferred. if we don't need to rewrite the binaries of the libraries, it would be great to avoid it. especially if the goal of this PR is to get rid of jetifier. (otherwise why are we doing this?) :)